docs_guide/others/general_pre-requisites.rst

```diff
@@ -0,0 +1,26 @@
+Pre-requisites
```

Comment:
I'm in two minds about this. When I wrote the pre-requisites I tried to make sure that they were minimal for each file, so I didn't tell people to create a `tutorials` folder when they wanted to add a how-to guide.

However I understand that this way the docs guide is easier to maintain, as if there were some structural changes to the documentation, we'd only need to change one file instead of most of the how-tos.

Nonetheless, if we commit to using general pre-requisites shouldn't we also include the rest of the sidebar (`getting_started.rst`, `release_notes.rst`)? But then if we decide to include all the sidebar, it may be easier to simply link to the tutorial in which all of this is covered (`intro/new_doc_project.rst`)

Reply:
Yes the main motivation for splitting it into a separate document and merging the different folder structure into only one for reducing the maintenance burden. But I see what you are getting at. I will link to the `intro/new_doc_project.rst` but I think it's still important to show the directory in the prerequisites because that's the key information and existing projects many not have the necessary folders. I will also rephrase that the folder structure is not minimal, but the folder of interest in the specific guide will be required. Do you think that's good enough?

Reply:
Done in [37f4cdb](https://github.com/Qiskit/qiskit_sphinx_theme/pull/166/commits/37f4cdbe1120a5e45b60044553100a1c2b84bae0)